content/pt-br/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -0,0 +1,277 @@
+Qualidade de Serviço (QoS). O Kubernetes usa classes QoS para tomar decisões sobre
+agendamento e despejo de Pods.
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+
+<!-- steps -->
+
+## Classes QoS
+
+Quando o Kubernetes cria um Pod, ele atribui uma dessas classes de QoS ao Pod:
+
+* Guaranteed
+* Burstable
+* BestEffort
+
+## Crie um namespace
+
+Crie um namespace, assim os seus recursos criados neste exercício estarão 
+isolados do resto do seu cluster.
+
+```shell
+kubectl create namespace qos-example
+```
+
+## Crie um Pod ao qual seja atribuída uma classe de QoS `Guaranteed`
+
+Para que um Pod receba uma classe de QoS `Guaranteed`:
+
+* Todo Contêiner no Pod deve ter um limite de memória e um requisito de memória.
+* Para cada Contêiner no Pod, o limite de memória deve ser igual ao requisito de memória.
+* Todo Contêiner no Pod deve ter um limite de CPU e um requisito de CPU.
+* Para cada Contêiner no Pod, o limite da CPU deve ser igual ao requisito da CPU.
+
+Essas restrições se aplicam a contêineres de inicialização e contêineres de aplicativos, igualmente.
+
+Aqui está o arquivo de configuração para um pod que possui um contêiner. O contêiner tem um limite de memória e um requisito de memória, ambos iguais a 200 MiB. O contêiner tem um limite de CPU e uma solicitação de CPU, ambos iguais a 700 milicpu:
+
+{{< codenew file="pods/qos/qos-pod.yaml" >}}
+
+Crie o Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/qos/qos-pod.yaml --namespace=qos-example
+```
+
+Veja informações detalhadas sobre o pod:
+
+```shell
+kubectl get pod qos-demo --namespace=qos-example --output=yaml
+```
+
+A saída mostra que o Kubernetes forneceu ao pod uma classe de QoS `Guaranteed`. A saída também
+verifica se o Contêiner do Pod tem um requisito de memória que corresponde ao seu limite de memória, e possui
+um requisito de CPU que corresponde ao seu limite de CPU.
+
+```yaml
+spec:
+  containers:
+    ...
+    resources:
+      limits:
+        cpu: 700m
+        memory: 200Mi
+      requests:
+        cpu: 700m
+        memory: 200Mi
+    ...
+status:
+  qosClass: Guaranteed
+```
+
+{{< note >}}
+Se um contêiner especificar seu próprio limite de memória, mas não especificar um requisito de memória, o Kubernetes
+automaticamente atribui um requisito de memória que corresponda ao limite. Similarmente, se um Contêiner especifica o seu próprio
+limite da CPU, mas não especifica um requisito de CPU, o Kubernetes atribui automaticamente uma solicitação de CPU que corresponde
+ao limite.
+{{< /note >}}
+
+Apague seu Pod:
+
+```shell
+kubectl delete pod qos-demo --namespace=qos-example
+```
+
+## Crie um Pod ao qual seja atribuída uma classe de QoS `Burstable`
+
+Um Pod recebe uma classe de QoS `Burstable` se:
+
+* O Pod não atende aos critérios para a classe de QoS `Guaranteed`.
+* Pelo menos um Contêiner no Pod tem um requisito ou limite de memória ou CPU.
```

Comment:
If writing the translated term “Contêiner”, don't capitalize the first letter.

Reply:
Hi, Really I'm not sure, when use "Contêiner(es)" capitalize or not, then I follow the original english docs, whithout questioning, it's a good opportunity to know why.

Reply:
when written with initial caps “Container” refers to the Container sub-API of Pod. We don't localize API names.